readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -0,0 +1,504 @@
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Ad-free documentation</div>
+                    <div class="description">stop showing ads to all your users on your Read the Docs hosted documentation</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+      </div>
+
+    </div>
+  </section>
+
+
+  <!-- Intro -->
+  <section class="ui vertical segment padded" id="intro">
+    <div class="ui container">
+
+      <div class="ui vertical segment basic padded">
+
+        <div class="ui grid center aligned">
+          <div class="column twelve wide computer sixteen wide mobile">
+
+            <h3 class="ui header massive"><span class="ui header">About our plans</span></h3>
+            <p>
+              Our plans are divided in two main types: the Community and the Enterprise plans.
+            </p>
+            <p>
+              The Community plans are meant for open source projects and do not allow private repositories or documentation. Here we have the base free plan and the gold plan. Additionally, the base plan will allow for donations to be made that will turn it into an Ad-free plan for a limited time, dependant on the donation amount and recurrence.
+            </p>
+
+          </div>
+        </div>
+
+      </div>
+
+      <!-- Enterprise Info -->
+      <div class="ui vertical segment basic very padded center aligned">
+        <h3 class="ui header massive">Enterprise plans</h3>
+        <p>
+          Offer flexible options for your organizations, whether hosting public or private documentation.
+        </p>
+      </div>
+
+      <!-- Enterprise Extra Info -->
+      <div class="ui vertical segment basic very padded">
+        <div class="ui grid stackable three column">
+
+          <div class="column">
+            <h4 class="ui header big">&emsp;</h4>
+            <h4 class="ui header">Additional features</h4>
+            <p>
+              For enterprise documentation needs not met by our other plans, we can offer customized plans with additional services. These include:
+            </p>
+            <ul>
+              <li>IP address allow lists to ensure only VPN users have access to your content</li>
+              <li>SLA-guaranteed support response times and documentation uptime</li>
+              <li>Unlimited audit tracking and analytics storage duration</li>
+              <li>Custom build requirements (additional memory, custom architectures)</li>
+            </ul>
+            <p>
+              We are happy to work with you on any additional requests you might have. Reach out with your specific needs.
+            </p>
+            <p>
+              Still Have Questions? <span class="ui segment basic tiny"><a href="" class="ui button black mini">Contact us</a></span>
```

Comment:
Missing a href here.

Reply:
We still don't have that page. Should leave the link to the future page or link to an email support for now?

```suggestion
              Still Have Questions? <span class="ui segment basic tiny"><a href="mailto:support@readthedocs.com" class="ui button black mini">Contact us</a></span>
```

Reply:
Yeah, `support@` is enough. I don't think we'd want to publish our individual contact information on a contact page, we just use the same mailbox for all of our inbound email.

Reply:
Great! Committing this change.